dhis-2/dhis-services/dhis-service-administration/src/main/java/org/hisp/dhis/appstore/DefaultAppStoreService.java

```diff
@@ -83,17 +83,7 @@ public DefaultAppStoreService( RestTemplate restTemplate, AppManager appManager,
     public List<WebApp> getAppStore()
     {
         String appStoreApiUrl = dhisConfigurationProvider.getProperty( ConfigurationKey.APP_STORE_API_URL );
-        String appStoreApiVersion = dhisConfigurationProvider.getProperty( ConfigurationKey.APP_STORE_API_VERSION );
-        String allAppsUrl;
-
-        if ( "" == appStoreApiVersion )
-        {
-            allAppsUrl = appStoreApiUrl + "/apps";
-        }
-        else
-        {
-            allAppsUrl = appStoreApiUrl + "/" + appStoreApiVersion + "/apps";
-        }
+        String allAppsUrl = appStoreApiUrl + "/v1/apps";
```

Comment:
remove hardcoded v1? Otherwise we cant specify v2 in config

Reply:
If we want to specify `v2` in the config, we might as well keep it separate like it was.

Reply:
There two are equivalent, neither gives the option for some end-points to stay on e.g. `v1` and some to use `v2`.

```
appstore.api.version = "v2"
appstore.api.url = "path/to/api"
```

and:

```
appstore.api.url = "path/to/api/v2"
```

If the core needs to use `v2/apps` then some code needs to change in core on how to parse the `v2` response anyway.

If the core needs to use `v2` for one end-point and `v1` for another, it needs the base API URL to be able to build e.g. `/api/v2/apps` and `/api/v1/app` at the same time.

Reply:
Actually, to remain backward compatible with the legacy app store this needs to default to an unversioned `/apps` end-point: 906c1fd8e